vignettes/vignette.Rnw

```diff
@@ -485,27 +494,31 @@ oncoprint(pathway, title = 'Custom pathway',  font.row = 8, cellheight = 15, cel
 \caption{\texttt{oncoprint} output of a custom pathway called {\tt MyPATHWAY} involving genes SETBP1, EZH2 and WT1; the genotype of each event is shown.}
 \end{figure*}
 
-In TRONCO there is also a function which creates the pathway view and the corresponding oncoprint to  multiple pathways, when these are given as a list. We make here a simple example of two custom pathways.
-
+In TRONCO there is also a function which creates the pathway view and the corresponding oncoprint to  multiple pathways, when these are given as a list. We make here a simple example of the next two pathways in our search result from WikiPathways.
 
 <<onco-pathway-viz, fig.show='hide', fig.width=6.5, fig.height=1.8>>=
+pw2.title<-my.hs.pathways[[2]]$name
+pw2.genes<-getXrefList(my.wpids[2],"H")
+pw3.title<-my.hs.pathways[[3]]$name
+pw3.genes<-getXrefList(my.wpids[3],"H")
+
+pathway.list<-list()
+pathway.list[[pw2.title]]<-pw2.genes
+pathway.list[[pw3.title]]<-pw3.genes
```

Comment:
Maybe could be good to show to the user what's inside this list.

Reply:
`pathway.list` is just a named list of pathway titles and their genes. That's pretty clear from the variable names. Do you mean to actually add a line that prints out the contents of `pathway.list`?  I can do that...